ao_stock/views/report_stockpicking_operations.xml

```diff
@@ -1,89 +1,116 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <template id="external_layout_header_ao"
-              inherit_id="stock.external_layout_header_barcode_right">
-        <xpath expr="//div[@class='header']" position="replace">
-            <div class="header">
-                <div class="row">
-                    <div style="margin-left:100px;text-align:left;">
-                        <img t-if="company.logo" t-att-src="'data:image/png;base64,%s' % company.logo" style="max-height: 40px;"/>
-                    </div>
-                    <div class="col-xs-9 text-right" style="margin-top:20px;" t-field="company.rml_header1"/>
-                </div>
-                <div class="row">
-                    <div style="margin-left:120px;text-align:left;">
-                        <div t-field="company.partner_id" t-field-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;, &quot;name&quot;], &quot;no_marker&quot;: true}"/>
-                    </div>
-                </div>
-            </div>
-        </xpath>
-    </template>
+    <!--<template id="external_layout_header_ao"-->
+              <!--inherit_id="stock.external_layout_header_barcode_right">-->
+        <!--<xpath expr="//div[@class='header']" position="replace">-->
+            <!--<div class="header">-->
+                <!--<div class="row">-->
+                    <!--<div style="margin-left:100px;text-align:left;">-->
+                        <!--<img t-if="company.logo" t-att-src="'data:image/png;base64,%s' % company.logo" style="max-height: 40px;"/>-->
+                    <!--</div>-->
+                    <!--<div class="col-xs-9 text-right" style="margin-top:20px;" t-field="company.rml_header1"/>-->
+                <!--</div>-->
+                <!--<div class="row">-->
```

Comment:
Why is this commented?

Reply:
Because this part doesn't work, this layouts doesn't exist. Instead of deleting them, I commented them, because maybe we will need to refactor this. I don't know, but maybe aleph wants a specific header or something.